gradoop-flink/src/main/java/org/gradoop/flink/io/impl/csv/tuples/CSVEdge.java

```diff
@@ -15,50 +15,89 @@
  */
 package org.gradoop.flink.io.impl.csv.tuples;
 
-import org.apache.flink.api.java.tuple.Tuple5;
+import org.apache.flink.api.java.tuple.Tuple6;
 
 /**
  * Tuple representing an edge in a CSV file.
  */
-public class CSVEdge extends Tuple5<String, String, String, String, String> {
+public class CSVEdge extends Tuple6<String, String, String, String, String, String> implements CSVElement {
 
+  @Override
   public String getId() {
     return f0;
   }
 
+  @Override
   public void setId(String id) {
     f0 = id;
   }
 
-  public String getSourceId() {
+  /**
+   * Returns the gradoop ids of the graphs that the edge belongs to.
+   * @return graph gradoop ids
+   */
+  public String getGradoopIds() {
     return f1;
   }
 
+  /**
+   * Sets the gradoop ids of the graphs which contain this edge.
+   * @param gradoopIds graph gradoop ids
+   */
+  public void setGradoopIds(String gradoopIds) {
+    f1 = gradoopIds;
+  }
+
+  /**
+   * Returns the string that represents the source.
+   * @return returns the source id.
+   */
+  public String getSourceId() {
+    return f2;
+  }
+
+  /**
+   * Sets the string that represents the source.
+   * @param sourceId represents the source id.
+   */
   public void setSourceId(String sourceId) {
-    f1 = sourceId;
+    f2 = sourceId;
   }
 
+
+  /**
+   * Returns the string that represents the target.
+   * @return returns the target id.
+   */
   public String getTargetId() {
-    return f2;
+    return f3;
   }
 
+  /**
+   * Sets the string that represents the target.
+   * @param targetId represents the target id.
```

Comment:
Please add an empty line between JavaDoc description and @param for all functions in this class.